core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1246,13 +1246,15 @@ class KafkaApis(val requestChannel: RequestChannel,
       }
 
     var clusterAuthorizedOperations = Int.MinValue
```

Comment:
`-2147483648` is actually `Int.MinValue`. It would cause an error if we would write something different from the default value for versions < 8 and > 10.

Reply:
Could you add the comments for that value (```Int.MinValue```) for dumb readers like me :)

Reply:
Will do.